docs/site/_includes/getting_started/existing/partials/TROUBLESHOOT_RU.liquid

```diff
@@ -0,0 +1,58 @@
+Кластеры некоторых провайдеров могут требовать дополнительных действий, необходимых для запуска Deckhouse и его компонентов (особенно при использовании managed-решений), например:
+    - узлы кластера могут иметь набор labels/taints которые не позволяют запускаться компонентам Deckhouse;
+    - в кластере могут работать validation-хуки, мешающие установке, и т.п.
+
+Ниже приведены некоторые такие случаи и частые проблемы. Если вы столкнулись с другими особенностями установки Deckhouse в существующем кластере, пожалуйста, опишите ваш опыт в [issue](https://github.com/deckhouse/deckhouse/issues).
+
+{% offtopic title="Ошибки на этапе 'Waiting for Deckhouse to become Ready'" %}
+- Ошибка вида:
+  ```
+  │ │ ┌ Waiting for Deckhouse to become Ready
+  │ │ │ Deckhouse pod found: deckhouse-64649df6f9-mf6dt (Pending)
+  │ │ │ Deckhouse pod found: deckhouse-64649df6f9-mf6dt (Pending)
+  ```
+
+  Возможная причина — в кластере нет узла с label `node-role.kubernetes.io/master: ""`, который первоначально используется в `nodeSelector` манифеста deployment `deckhouse`.
+
+  Варианты решения:
+  - Отредактировать deployment `deckhouse` и указать нужный nodeSelector:
+    ```shell
+    kubectl -n d8-system edit deployment/deckhouse
+    ```
+  - Удалить nodeSelector у deployment `deckhouse`:
+    ```shell
+    kubectl patch -n d8-system deployment deckhouse --type json -p '[{"op": "remove", "path": "/spec/template/spec/nodeSelector"}]'
+    ```
+{% endofftopic %}
+
+{% offtopic title="Кластер в VK Cloud Solutions (MailRu Cloud Solutions)" %}
+<ul><li><p>В кластерах версии 1.21+ присутствует Gatekeeper (OPA), который требует выставления limits и requests подам. Pod Deckhouse архитектурно не имеет limits/requests, и, после установки Deckhouse, его Pod не будет запущен. При просмотре событий Deployment <code>deckhouse</code> вы можете увидеть ошибку:</p>
```

Comment:
I think, removing Gatekeeper is not good idea.

Gatekeeper can protect another infrastructure or managed k8s components. An we can break somethings in cluster managed components or open secure holes. Or client can use gatekeeper for its own goals.

We can add resource limits with special arguments and add parameters to deckhouse module?
Can we configure gatekeeper and add excluded rules for Deckhouse?

@nabokihms What do you thinks about it?

Reply:
Gatekeeper forces users to set requests/limits on all components and conflicts with our VPA service. That's why Artem added this step.

Reply:
The section was updated. Need review.